app/Http/Controllers/Authcontroller.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+
+class Authcontroller extends Controller
+{
+
+    /**
+     * Show the form for creating a new resource.
+     */
+    public function register(Request $request)
+    {
+        $validatedData = $request->validate([
+            'name'=> 'required|string|max:255',
+            'email'=>'required|email|string|unique:users',
+            'password'=> 'required|string|min:8|confirmed'
+        ]);
+
+        $user = User::create([
+            'name'=> $validatedData['name'],
+            'email'=> $validatedData['email'],
+            'password' => Hash::make($validatedData['password']),
+        ]);
```

Comment:
Could you please tell me why you'll redirect user to login after account creation?

Instead do Auth::login($user);

Reply:
Solved